### PR TITLE
Samples: Update sample to NET 6

### PR DESCRIPTION
**Pull request by Venkat1340**
## Purpose

* ... The goal is to update the project in that repo to NET 6 (currently in NET Core 3.1)

## Does this introduce a breaking change?

```
[ ] Yes
[X ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[X ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
from .Net Core 3.1 to .Net core 6 upgrade is there or not.
* ...

### CODE REVIEW

File: CosmosGettingStartedTutorial/CosmosGettingStarted.csproj [hunk 1]
```diff
@@ -11,6 +11,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Cosmos" Version="*" />
+    <PackageReference Include="Microsoft.AspNetCore.JsonPatch" Version="6.0.0" />
```
Comment (ealsur) [inline CosmosGettingStartedTutorial/CosmosGettingStarted.csproj:14]: Why are these changes required? Can you try to remove these packages?

Comment (Venkat1340) [reply CosmosGettingStartedTutorial/CosmosGettingStarted.csproj:14]: added looking at the 'Update package references'. from documentation. after remove all those new packages added. the project is built successfully.

**Review by ealsur — APPROVED** [review]
